src/strings/PowerFxResources.en-US.resx

```diff
@@ -4760,4 +4760,8 @@
     <value>Unsupported untyped/JSON conversion type '{0}' in argument.</value>
     <comment>Error Message shown to user when a unsupported type is passed in type argument of AsType, IsType and ParseJSON functions.</comment>
   </data>
+  <data name="ErrReachedMaxJsonDepth" xml:space="preserve">
```

Comment:
>ErrReachedMaxJsonDepth [](http://example.com/codeflow?start=14&length=22)

This is a runtime error message, so it should go to the resources for runtime, not the one for core.

Reply:
We'd have to put in Json assembly as we can't access Interpreter resources
If that's not a huge deal for now, I'd keep it in Core

Reply:
[https://github.com/microsoft/Power-Fx/issues/2646](https://github.com/microsoft/Power-Fx/issues/2646)